upload/src/addons/SV/WarningImprovements/_output/templates/public/alert_warning_alert_warning.html

```diff
@@ -1,16 +1,13 @@
-<xf:if is="{$user.user_id}">
-	{{ phrase('alert_profile_post_insert', {
-			'username': username_link($user, false, {'defaultname': $alert.username}),
-			'warningLinkAttributes': 'href="' . link('warnings', $content) . '" class="fauxBlockLink-blockLink"',
-			'points': {$content.points},
-			'warning_title': {$content.title}
-		})
-	}}
-<xf:else/>
-	{{ phrase('warning_alert_text_anonymize', {
-			'warningLinkAttributes': 'href="' . link('warnings', $content) . '" class="fauxBlockLink-blockLink"',
-			'points': {$content.points},
-			'warning_title': {$content.title}
-		})
-	}}
-</xf:if>
\ No newline at end of file
+<xf:set var="$issuer" value="{$alert.User}" />
```

Comment:
This isn't really effective, as the sender username/avatar is burnt into the displayed alert which the template can't influence.

Reply:
That replaced using template modification :)

Reply:
Yup, you are right!